app/views/admin/trends/tags/_tag.html.haml

```diff
@@ -10,7 +10,8 @@
 
       %br/
 
-      = t('admin.trends.tags.used_by_over_week', count: tag.history.reduce(0) { |sum, day| sum + day.accounts })
+      = link_to tag_path(tag) do
```

Comment:
Sorry I'm just noticing and suggesting a change now, but I believe this would be more consistent if it opened in a new tab:
```suggestion
      = link_to tag_path(tag), target: '_blank' do
```